test/methods.js

```diff
@@ -2,280 +2,285 @@
 'use strict';
```

Comment:
I think the main whitespace change here is that we went from 2 space tabs to 4 space tabs. Would you mind changing the file back to 2 space tabs to avoid this discrepancy? I realize now that the module is internally inconsistent. I personally prefer 2 space tabs and use that as my editor settings. There is not a style guide yet (though there should be for issues such as this). But at least in the meantime let's keep individual files consistent.